albumentations/augmentations/transforms.py

```diff
@@ -214,6 +226,35 @@ def get_transform_init_args_names(self):
             "mask_value",
         )
 
+    def __update_position_params(
+        self, h_top: int, h_bottom: int, w_left: int, w_right: int
+    ) -> Tuple[int, int, int, int]:
+        if self.position == "top_left":
+            h_bottom += h_top
+            w_right += w_left
+            h_top = 0
+            w_left = 0
+
+        if self.position == "top_right":
+            h_bottom += h_top
+            w_left += w_right
+            h_top = 0
+            w_right = 0
+
+        if self.position == "bottom_left":
+            h_top += h_bottom
+            w_right += w_left
+            h_bottom = 0
+            w_left = 0
+
+        if self.position == "bottom_right":
+            h_top += h_bottom
+            w_left += w_right
+            h_bottom = 0
+            w_right = 0
```

Comment:
I think it better to use `elif` constraction there.

Reply:
@Dipet fixed, can you check, please?